projects/microphysics/experiments/default.py

```diff
@@ -0,0 +1,21 @@
+import sys
```

Comment:
Would be nice to have this in a cron job so we could detect performance regressions faster.

Reply:
Agree, I can cook one up after this merges.